utils/python/CIME/build.py

```diff
@@ -237,6 +240,9 @@ def case_build(caseroot, case, sharedlib_only=False, model_only=False):
     os.environ["CISM_USE_TRILINOS"]    = stringify_bool(cism_use_trilinos)
     os.environ["MPASLI_USE_ALBANY"]    = stringify_bool(mpasli_use_albany)
 
+    if get_model() == "acme" and mach == "titan" and compiler == "pgiacc":
```

Comment:
Ultimately yes but we needed a way, at create_newcase time, to set the CAM source path based on machine and compiler.   We didn't want to add another create_newcase argument.